src/components/InvoicesList/InvoicesList.js

```diff
@@ -56,21 +74,17 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
       </Helmet>
       <HeaderSection>
         <div className="col-sm-12 col-md-12 col-lg-12">
-          <nav className="dp-breadcrumb">
-            <ul>
-              <li>
-                <a href={_('invoices_list.control_panel_account_preferences_url')}>
-                  {_('invoices_list.control_panel_section')}
-                </a>
-              </li>
-              <li>
-                <a href={_('invoices_list.control_panel_billing_information_url')}>
-                  {_('invoices_list.control_panel_billing_information_section')}
-                </a>
-              </li>
-              <li>{_('invoices_list.title')}</li>
-            </ul>
-          </nav>
+          <Breadcrumb>
+            <BreadcrumbItem
+              href={_('invoices_list.control_panel_account_preferences_url')}
+              text={_('invoices_list.control_panel_section')}
+            />
+            <BreadcrumbItem
+              href={_('invoices_list.control_panel_billing_information_url')}
+              text={_('invoices_list.control_panel_billing_information_section')}
+            />
+            <BreadcrumbItem text={_('invoices_list.title')} />
```

Comment:
Acá vas a tener que implementar alguna lógica para no hacer un `a` cuando no tiene `href`

Reply:
ahh, veo que ya lo hiciste!  ❤️

Reply:
Yes, I added the following:

```
const BreadcrumbItem = ({ href, text }) => (
    <>
      {!!href ? (
        <li>
          <a href={href}>{text}</a>
        </li>
      ) : (
        <li>{text}</li>
      )}
    </>
  );
```

Reply:
Guarda que esto de los breadcrumb esta en varios lugares, la idea seria que sea un componente reusado por Shopify, y reportes creo que es el otro componente, e invoices.

Reply:
Y esta solución no es la mejor para ser reusada? desacopla completamente el breadcrom del contenido

Reply:
Según lo que vi se usa en: AgenciesForm, Shopify, InvoicesList y Subscribers. Se podría ver de mover el componente de breadcrumb que esta en Invoices a shared y empezar a usar ese componente. No se que les parece.

Reply:
En un futuro, esto se puede hacer más inteligente, que el breadcrumb dependa del ruteo y genere las secciones y los links automáticamente, pero yo dejaría eso para un futuro, para que lo agarre alguien del equipo de webapp.

Reply:
Sisi, obvio, mi idea es simplemente mover lo que tengo en invoices a un componente en shared y usarlo en invoices y despues que el equipo de webapp hago todos los cambios que vean convenientes. Si me dan el Ok me pongo con eso.

Reply:
Sisi me parece genial, esa era la idea, lo veo bien al breadcrumb.

Reply:
Cuando hagas el componente podras sumar algun test?

Reply:
Sisi, obvio, ya había pensado en crear la clase de test

Reply:
I added the new component and I used it in the invoices pages and also added some unit tests.

Reply:
Great!